arquillian-chameleon-runner/runner/src/main/java/org/arquillian/container/chameleon/runner/RunnerExpressionParser.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2016 pact-jvm project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.arquillian.container.chameleon.runner;
+
+import java.util.StringJoiner;
+
+public class RunnerExpressionParser {
```

Comment:
Missing test for this one too. (and I think we might have similar thing in `-core`)